### DISCUSSION TITLE
Looking for an explanation of the legend

### DISCUSSION THREAD

**Discussion by talhaanwarch** [cdpierse/transformers-interpret#138] [Q&A]
In the Sequence Classification Explainer and Pairwise Sequence Classification section, does legend represent the class name?
let say i have two classes named as "low quality and high quality" as shown
![image](https://github.com/cdpierse/transformers-interpret/assets/37379131/a38b889d-4fc7-4c3a-bdfb-a081adcf6c06)
how can i interpret legends and color in this image
or may in this image
![image](https://github.com/cdpierse/transformers-interpret/assets/37379131/b31863d7-2d18-458b-bea4-ed288ff11cdf)

i am having issues in understanding these color meanings
What it mean if more of words are green or red?

**Comment by talhaanwarch** [comment]
my understanding is that if positive color is more it mean that words contributed more in defining that attribution class, no matter what class it is?

**Comment by David-Saeteros** [comment]
Can you give an example where the Predicted and Attribution Label are both low_quality?
